_includes/pbc/all/install-features/202311.0/install-the-service-points-order-management-feature.md

```diff
@@ -111,6 +314,7 @@ class ShopApplicationDependencyProvider extends SprykerShopApplicationDependency
 
 Make sure that the following widgets have been registered by adding the respective code snippets to a Twig template:
```

Comment:
How do I validate the result after adding the the widget?

Reply:
If the widget is added you should be able to see the service point name within order details as soon as the order is related to a service point. Example can be found as screenshot in [CC-26584](https://spryker.atlassian.net/browse/CC-26584).